Wikipedia/Code/MWKDataStore.m

```diff
@@ -461,6 +461,8 @@ - (void)performUpdatesFromLibraryVersion:(NSUInteger)currentLibraryVersion inMan
     
     if (currentLibraryVersion < 14) {
         [self.remoteNotificationsController deleteLegacyDatabaseFilesAndReturnError:nil];
+        NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
+        userDefaults.wmf_shouldShowNotificationsExploreFeedCard = YES;
         [NSHTTPCookieStorage migrateCookiesToSharedStorage];
```

Comment:
As mentioned below, let's also add this line to delete any old notifications cards from their database, to keep our content source logic simple:

```
[moc removeAllContentGroupsOfKind:WMFContentGroupKindNotification];
```